### PR TITLE
Fix disclaimer bug

### PR DESCRIPTION
**Pull request by gabrielbazan7**

### CODE REVIEW

**Comment by gabrielbazan7** [comment]
update. i'm still not convinced of the way to solve the last issue (fix blank page) @cmgustavo

File: src/js/controllers/disclaimer.js [hunk 1]
```diff
@@ -32,6 +36,8 @@ angular.module('copayApp.controllers').controller('disclaimerController',
       });
     };
 
-    create();
-
+    if (!profileService.profile)
+      create();
+    else
+      applicationService.restart();
```
Comment (cmgustavo) [inline src/js/controllers/disclaimer.js]: This action will restart Copay while I just want to view the disclaimer from Settings->About->Disclaimer.

Comment (gabrielbazan7) [inline src/js/controllers/disclaimer.js]: Yes. That problem not solved yet. I will make a new file that includes only the Terms of Use and not the icon and "welcome to copay" title.

**Comment by gabrielbazan7** [comment]
updated

**Comment by gabrielbazan7** [comment]
Conflicts resolved.

**Comment by cmgustavo** [comment]
I found a specific issue: when I delete manually the flag `agreeDisclaimer` from localStorage, Copay delete all my current wallets and creates a new one.

**Comment by cmgustavo** [comment]
For slow connections (or devices), I would like a spinner or something that indicating that Copay is loading/creating a wallet

**Comment by cmgustavo** [comment]
iOS: Preferences -> About Copay -> Terms of Use

I can not scrolling the page down.

**Comment by gabrielbazan7** [comment]
updated

**Comment by gabrielbazan7** [comment]
updated

**Comment by gabrielbazan7** [comment]
updated

**Comment by cmgustavo** [comment]
ACK

**Pull request merged**